%% The following is a directive for TeXShop to indicate the main file
%%!TEX root = diss.tex

%% 350 words
\chapter{Abstract}

Cancer is an ecosystem of genetically diverse evolving clones, which emerge in time and space as a result of genomic and non-genomic instability. Consequently, clonal evolution provides a basis for fluctuating cell fitness, which impacts etiology and drug resistance. However, progress in defining clonal fitness by copy number alterations (CNA), has been impeded by a lack of perturbation experiments with timeseries sampling. This dissertation is focused on understanding and measuring clonal fitness determined by copy number genomic instability in breast cancer to single cell resolution. First, I established a series of transplantable human breast cancers in immunodeficient mice strains that were tolerant to DNA damaging chemotherapy. I selected four PDXs (patient derived xenografts) for detailed analysis of clonal fitness under no treatment and after treatment with DNA damaging agents, cisplatin and CX5461. Subsequently, I optimized methods of tumor dissociations, and determined the effects of digestion time and temperature on single cell gene expression. Next, I established that single cell whole genome sequencing of CNA in PDX passaged over time resulted in positive fitness over clones that gradually sweep the population. I established a mixture re-transplant paradigm to demonstrate that the fitness predictions of a Wright-Fisher population genetics model applied to the data, were reproducible experimentally.  I described that fitness landscapes of all TNBC tumors were inverted under drug, meaning that low fitness clones under no treatment gave rise to drug resistant clones. In drug holiday experiments, I showed that when clonal competition was still possible during early drug resistance, cisplatin resistance had a fitness cost. I demonstrated using CX5461, that drug resistance could arise in a common background, suggesting common drug resistant states. Finally, I examined the impact of CNA on transcriptional phenotypes. By making assignments of scRNA-seq single cells, to CNA defined clones, I observed that in cis, CNA mediated clonal gene expression impacts between 5-50\% of the transcriptome. Time series analysis of expression revealed reversible, but time dependent expression reprogramming, thus defining the non-CNA dependence of clonal transcriptomes. Pathway analysis revealed common sets of pathways associated with late drug resistance to platinum in TNBC.

These comprehensive measurements and analysis of clonal structure will advance interpretation of polyclonal resistance to therapy.


















%Cancer is an ecosystem of genetically diverse evolving clones, which emerge in time and space as a result of genomic and non-genomic instability. Consequently, clonal evolution provides a basis for increased and decreased cell fitness, which impacts etiology and drug resistance. However, progress in defining clonal fitness landscapes, especially those defined by copy number alterations (CNA), has been impeded by a lack of controlled perturbation experiments with timeseries single cell sampling of polyclonal populations at single cell resolution. This dissertation is focused on understanding and measuring clonal fitness determined by copy number genomic instability in breast cancer to single cell resolution.
%on the development of serial passaging of TP53 mutant human breast cancers, including three triple negative breast cancer (TNBC) patient derived xenografts (PDXs) timeseries, with and without drug perturbations. Our goal is to reveal the intricacies of cancers to explore the effects of spatial and temporal heterogeneity on the population dynamics.
 %First, I established a series of transplantable human breast cancers in immunodeficient mice strains that are tolerant to DNA damaging chemotherapy. I selected four PDX lines for detailed analysis of clonal fitness under no treatment and after treatment with DNA damaging agents, cisplatin and CX5461. Subsequently, I optimized methods of tumor dissociations, and determined the effects of digestion time and temperature on single cell gene expression. Next, I established that single cell whole genome sequencing of CNA in PDX passaged over time, resulted in a spread of weakly positive fitness over clones that gradually sweep the population. I established a mixture re-transplant paradigm to demonstrate that the fitness predictions of a Wright-Fisher population genetics model applied to the data, were reproducible experimentally. Next, I described that by continuous exposure to cisplatin, tumor fitness landscapes of all TNBC tumors were inverted, meaning that low fitness clones under no treatment gave rise to drug resistant clones and this was also reproducible. In drug holiday experiments, I showed that when clonal competition is still possible during early drug resistance, cisplatin resistance has a fitness cost. I showed using a drug of a different class, CX5461, that drug resistance can arise in a common background with different drugs, suggesting there may be common drug resistant states. Finally, I examined the impact of CNA on transcriptional phenotypes. By making assignments of scRNA-seq single cells, to CNA defined clones, I observed that in cis, copy number mediated clonal gene expression impacts between 5-50\% of the transcriptome. Time series analysis of expression revealed reversible, but time dependent expression reprogramming, thus defining the non-CNA dependent of clonal transcriptomes. Pathway analysis revealed common sets of pathways associated with late drug resistance to platinum in TNBC.

 %the tumor quatitative fitness model developed to ascribe selection coefficients to individual CNA defined clones, I found that
%Then you can describe that 
%under cisplatin, the fitness landscape was inverted, meaning that low fitness clones under no treatment gave rise to drug resistant clones and this was also reproducible. % Next, by using quatitative fitness model developed to ascribe selection coefficients to individual CNA defined clones, I found that in three different serially passaged cisplatin-treated patient derived xenograft (PDX) transplant lines of primary TNBC, the untreated fitness landscape was inverted. 
 
 %Notably, drug resistant clones reproducibly emerge from phylogenetic lineages of low fitness clones, and high fitness clones were eradicated. Moreover, drug treatment `holiday' time points indicate reversibility, suggesting both a fitness cost of the drug resistant genotypes and association of fitness inversion with drug selection.
 %Finally, I presented the close correlation between copy number variation and differential gene expression, by exploring single cell RNA sequencing data on the same independent serial samples. I found that majority of genomic variations generated a direct effect on gene transcriptional level and were clustered in chemotherapy resistance related pathways, which suggests the involvement of CNA in pathophysiology of drug resistance.

%These findings define approaches to comprehensive measurements and analysis of clonal structure in heterogeneous tumors and linking clonal fitness to CNA, and its related transcriptional changes, therefore, advance interpretation of polyclonal resistance to therapy.



% Consider placing version information if you circulate multiple drafts
%\vfill
%\begin{center}
%\begin{sf}
%\fbox{Revision: \today}
%\end{sf}
%\end{center}
