\chapter{Supporting Materials}

\begin{figure}
\centering
\includegraphics[width=\textwidth]{Figures/mutationanalysisbreakpoints.pdf}
  \caption[Structural variant and mutation rates of HER2+ and TNBC PDX]
	{\small
	\textbf{Structural variant and mutation rates of HER2+ and TNBC PDX.}
	    \textbf{(a)} Distribution over copy number
breakpoints/cell as a function of generation for left: TNBC, right: HER2+
   \textbf{(b)} Clone specific distributions over copy number breakpoints/cell, coloured by fitness coefficients for left: TNBC, right: HER2+
    \textbf{(c)} Clone specific distributions over point mutations/cell, coloured by fitness coefficients for left:TNBC, right:HER2+
}
    \label{fig:mutationanalysisbreakpoints}
    \end{figure}



\begin{figure}
\centering
  \includegraphics[width=\textwidth]{Figures/fig3_In_cispercentage.pdf}
	
\caption[Proportion of \textit{in cis} and \textit{in trans} regulated gene expression in scRNAseq data]
	{\small
	\textbf{Proportion of \textit{cis} and \textit{trans} regulated gene expression in scRNAseq data.}
	   Horizontal axis shows differential expression between two selected clones from all the three PDX treated and un-treated timeseries. Vertical axis gives percentage presence of genes \textit{in cis} or in trans. red bars represent in cis and blue bars represent in -trans regulated gene expression.
	   \textbf{(Upper)} This graph gives us percentage of genes by looking into the gene data set from COSMIC cancer gene dataset \cite{vogelstein2013cancer} that corresponds to our clone aware \ac{DE}.
	    \textbf{(Middle)} same like upper plot but looking into CORE FITNESS data set \cite{behan2019prioritization}.
	     \textbf{(Lower)} Same like above but looking into cisplatin related gene list curated from literature.
	    
	}
	\label{fig:fig3_In_cispercentage}
\end{figure}















\begin{figure}
\centering
\includegraphics[width=\textwidth]{Figures/treatedtimeseriesmanuscript.pdf}
  \caption[TNBC PDX timeseries clonal dynamics under drug perturbations]
	{\small
	\textbf{TNBC PDX timeseries showing timepoint nodes used for single cell whole genome sequencing}
	     All nodes representing each PDX tumour were digested to acquire genomes of single cells (~200-600 cells/tumor). Extra replicate tumors at each time point are not shown in the diagram (n=2-4). Grey circles represent un-treated, blue represents Cisplatin treated and grey with blue outline presents drug-holiday samples 
	     \textbf{(a)} SA609-TNBC time series with replicates. DLP+ collected starting from X1 to X10 (Un-Rx line 1). Top grey branch indicating Un-Rx line 2. The middle three branches are cisplatin treated time series replicate branches 
	     \textbf{(b)} SA535-TNBC  showing the tumor nodes taken for DLP+ starting from X5 untreated  \textbf{(c)} SA1035-TNBC  showing the tumor nodes taken for DLP+ starting from X 4 untreated.}
     \label{fig:treatedtimeseriesmanuscript}

\end{figure}




\begin{figure}
\centering
\includegraphics[width=\textwidth]{Figures/drugholidaysamples.pdf}
	
\caption[TNBC-SA609 PDX reproducible clonal dynamics with and without treatment]
	{\small
\textbf{Observed trajectories including drug-holiday samples.}
Panels showing line trajectories of clonal behaviour over time in two additional TNBC all three conditions. Fitness reversal in drug holiday samples of SA1035 and SA535 were not very prominent as compare to SA609. Horizontal axis representing the timepoints of each generation and vertical axis shows clonal fractions.
	}
	\label{fig:drugholidaysamples}
\end{figure}



\begin{figure}
\centering
\includegraphics[width=\textwidth]{Figures/SA1035_AllCyclesCisplatin.pdf}
	
\caption[Tumor growth curves from SA1035 TNBC PDX]
	{\small
\textbf{Tumor growth curves from SA1035 TNBC PDX.}
Tumour response curves in each cycle of cisplatin treatment in SA1035 TNBC PDX. The vertical axis on right indicates tumor status and on left shows tumor volume. Horizontal axis presents days post measurable tumor.
}
	\label{fig:SA1035_AllCyclesCisplatin}
\end{figure}


 % Table generated by Excel2LaTeX from sheet 'Sheet1'
 \begin{table}[htbp]
   \centering
   \caption{Cisplatin related genes from last 10 years literature}
     \begin{tabular}{llllr}
     \multicolumn{4}{l}{Gene IDs} &  \\
     VCAM1  & GCS    & BRCA2  & XAF1   &  \\
     VIM    & GST    & VDAC   & DYRK1B &  \\
     SLC31A1 & MT     & BAX    & ERBB2  &  \\
     GST    & ERCC1  & BCL    & HSPs   &  \\
     HMGB   & MLH1   & BIRC5  & TMEM205 &  \\
     CTR1   & MSH2   & CPN    & PDGFR &  \\
     ATP7A  & POLH   & CASP   & IGF1R  &  \\
     ATP7B  & REV3   & MAPKs  & TRP14  &  \\
     MRP2   & REV7   & p63    & RAB7   &  \\
     GSH    & BRCA1  & TP53   & RAB8   &  \\
     \end{tabular}%
   \label{tab:Cisplatinrelatedgenes}%
 \end{table}
 
 
 
 
 

\begin{figure}
\centering
\includegraphics[width=\textwidth]{Figures/Appendixtable.pdf}
\caption[Summary of number of genes \textit{in-cis} and \textit{in-trans}]
	{\small
	\textbf{Summary of number of genes \textit{in-cis} and \textit{in-trans}.}
	  Summary of number of genes up and down regulated within our data set with respect to the reference genes.

}
    \label{fig:Appendixtable}
    \end{figure}
    
   
   
 