%%%%%%%%
\chapter{Methods and Materials}

\section{Ethical approval}
The Ethics Committees at the University of British Columbia approved all the experiments using human resources. Patients in Vancouver, British Columbia, undergoing excision or diagnostic core biopsy were recruited, and samples were collected after written consent, under the tumour tissue repository (TTR-H06-00289) protocol and transplanted in mice under the Neoadjuvant patient derived xenografts (PDX) (University of British Columbia BC Cancer Research Ethics Board H20-00170) protocols. All experimental methods comply with the Helsinki declaration. All animal studies were approved by the Animal Care Committee at the University of British Columbia.  


\section{Establishment of patient derived xenografts} 
Patient's tumour materials were processed as described in \cite{eirew2015dynamics} and transplanted in mice under the \ac{ARC} bioethics protocol (A19-0298-A001) approved by the animal care committee.
Briefly, tumour fragments were chopped finely with scalpels and mechanically disaggregated for one minute using a Stomacher 80 Biomaster (Seward Limited, Worthing, UK) in \SIrange{1}{2}{\ml} cold DMEM/F-12 with Glucose, L-Glutamine and HEPES (Lonza 12-719F).
An aliquot of \SI{200}{\ul} of medium (containing cells/organoids) from the resulting suspension was used equally for 4 transplantations in mice.

Tumours were transplanted in mice as previously described~\cite{eirew2015dynamics} in accordance with SOP BCCRC 009. 
Briefly, female immuno-compromised, NOD/SCID/IL2r$\gamma^{\small{-/-}}$\ac{NSG} and        
\ac{NRG} \cite{pearson2008non} mice were bred and housed at the animal resource centre \ac{ARC} at the British Columbia (BC) Cancer Research Centre . 
For subcutaneous transplants, mechanically disaggregated cells and clumps of cells were re-suspended in \SIrange{150}{200}{\ul} of a 1:1 v/v mixture of cold DMEM/F12: Matrigel (BD Biosciences, San Jose, CA, USA).
8-12 weeks old mice were anesthetized with isofluorane, then the mechanically disaggregated cells/clumps suspension was injected under the skin on the left flank using a \SI{1}{\ml} syringe and 21 gauge needle. 
The animal care committee and animal welfare and ethical review committee, the University of British Columbia (UBC), approved all experimental procedures.

\subsection{Histopathology of PDX tumours}

\subsubsection{HER2+SA532 and TNBC-SA609-Line 1 in long untreated timeseries immunohistochemistry}
The hormone receptor status of the tumour samples were determined by immunohistochemistry and FISH (Fluorescence in situ hybridization) copy number.
Two separate tissue microarrays were prepared using duplicate \SI{1}{\mm} cores extracted from formalin-fixed paraffin-embedded blocks containing material from all passages of the patient derived xenografts (TNBC-SA609 and HER2+SA532) used for long un-treated timeseries. 
Deparaffinized \SI{4}{\um} sections of paraformaldehyde fixed tumours were processed for immunohistochemistry (IHC) using a Discovery XT automated system (Ventana Medical Systems, Tucson, AZ, USA). 
EGFR, INPP4B, Ki67, PR, ECAD were all performed on the Ventana Discovery XT platform using CC1 for antigen retrieval, incubating for one hour at room temp, and using the UltraMap DAB detection kit.
Primary antibodies to ER$\alpha$ (clone SP1, Ventana), HER2 (clone 4B5, Ventana), EGFR (clone EP22, Epitomics, Burlingamen, CA, USA) and Ki67 (clone SP6, Thermo Scientific) 
Horseradish peroxidase-conjugated Discovery Universal Secondary Antibody (Ventana) was then applied and the slides developed using 3,3'-diaminobenzidine Map Kit (Ventana). 
The slides were reviewed by a pathologist. 

Early and late passages from SA532HER+ and TNBC-SA609 PDX  maintained their hormone receptor negative statuses (ER and PR =0) and strong (3+) for SMA and Ki67. SA532 retained its HER2+ (1+) status both in early and late passages.
\textbf{(\autoref{fig:HistologyearlylateSA532SA609})}. Hence, confirming that NRG PDX system retain architecture and receptor markers from early to late passages.

 \begin{figure}
\centering
\includegraphics[width=\textwidth]{Figures/chap2/HistologyearlylateSA532SA609.png}
	
\caption[Representative patterns of expression on histology in early and late passages]
	{\small
\textbf{Representative patterns of expression on histology in early and late passages.}
 \textbf{(a)} IHC of HER2+ tumours at passage 1 (X1) and passage 10 (X10), 4x and 20x (insets). Scale bars \SI{500}{\micro\metre} and \SI{100}{\micro\metre} (insets). On top of each panel presenting the antibody name and the right bottom square is showing the score of the stain.
\textbf{(b)} IHC of TNBC SA609 tumours at passage 2 (X2) and passage 10 (X10), 4x and 20x (insets). Scale bars \SI{500}{\micro\metre} and \SI{100}{\micro\metre} (insets). On top of each panel presenting the antibody name and the right bottom square is showing the score of the stain.}
	\label{fig:HistologyearlylateSA532SA609}
\end{figure}


Sections were deparaffinized in xylene, rehydrated in graded alcohol, and used for histology and immunostaining. 
H \& E and \ac{IHC}  of the two PDX tumours showed that TNBC is  derived from a triple negative breast cancer patient while HER2+ was derived from a HER2+ breast cancer patient. HER2 IHC was scored as 2+, HER2/CEP17 ratio was calculated as 6.5 (positive) \cite{ahn2020her2} from  \ac{FISH}  and focal high level amplification (average copy number state of 10) of the \ac{ERBB2} locus (approximately Chr17:37500001-38000000) was found in the DLP+ data.

%\subsubsection{TNBC-SA609-Line 2, TNBC-SA1035 and TNBC-1035}
%For the experiments in chapter 4 and 5, we re transplanted TNBC-SA609, and conducted drug treatment experiments along with TNBC-SA1035 and TNBC-SA535. The preparation of tissue paraffin sections are identical as explained in the previous section. 

%A new TMA was prepared from these tumors for \ac{IHC} staining for the markers ER, PR, HER2, CK5/6, CK14, SMA, E-Cad and Vimentin.  \textbf{(\autoref{fig:HistologySA609}},  \textbf{(\autoref{fig:HistologySA1035}}, \textbf{(\autoref{fig:HistologySA535})}. IHC scores are shown in the figures confirming their triple negative receptor status, however, the replicate tumor sections and different passages were also stained and confirmed to retain the receptors and biomarkers status in \textbf{\autoref{tab:TNBCTMAscoringIHC}.}


%-----------------------------------------------------------------

\begin{table}
\centering
\caption{List of antibodies and experimental conditions for staining TMAs for IHC.}


\label{stab:antibodieslist}
\begin{tabular}{|rl|l|l|l|}
  \hline
 & Specimen & Stain/IHC & Vendor / Ab Clone & Dilution  \\ 
  \hline
1 & TMA1 \& 2 & CK14 & Empire Genomics clone LL002 & 1 in 50 \\ 
  2 & TMA1 \& 2 & Ck5/6 & Dako D5/16 B4 & *RTU  \\ 
  3 & TMA1 \& 2 & Ck8 (CAM5.2) & BC Bioscience CAM5.2 & 1 in 10  \\ 
  4 & TMA1 \& 2 & EGFR & Epitomics 1902-1 & 1 in 100  \\ 
  5 & TMA1 \& 2 & ER & Ventana  clone SP1 & *RTU\\ 
  6 & TMA1 \& 2 & H\&E & *N/A & *N/A \\ 
  7 & TMA1 \& 2 & INPP4B & Abcam EPR3108Y ab81269 & 1 in 50 \\ 
  8 & TMA1 \& 2 & Ki67 & Abcam ab16667  & 1 in 400  \\ 
  9 & TMA1 \& 2 & PR & Abcam ab30285 & 1 in 200  \\ 
  10 & TMA1 \& 2 & Slug/Snail & Abcam ab85936 & 1 in 125 \\ 
  11 & TMA1 \& 2 & SMA & Dako clone 1A4 & 1 in 100 \\ 
  12 & TMA1 \& 2 & Trichrome & Sigma-HT15-1KT & *N/A \\ 
  13 & TMA1 \& 2 & Twist & NB120-49254 & 1 in 200\\ 
  14 & TMA1 \& 2 & Vimentin & Dako V9 & RTU  \\ 
  15 & TMA1 only & E-Cad & Cell Signal 3195 & 1 in 100  \\ 
  16 & TMA1 only & HER2 & Roche 4B5 & 1 in 8  \\ 
  17 & TMA2 only & E-Cad & Dako NCH-38 & *RTU  \\ 
  18 & TMA2 only & HER2 & Ventana  clone 4B5 & *RTU \\ 
   \hline
\end{tabular}%


\small\textbf{(*RTU: Ready to use; N/A: not applicable; TMA: Tissue microarray)}.

\end{table}
%-------------------------------------------------------------------------------


\subsection{Serial passaging of PDX}
Tumours were serially passaged as  described \cite{eirew2015dynamics}.
Briefly, for serial passaging of PDX, xenograft-bearing mice were euthanized when the size of the tumours approached \SI{1000}{\mm\cubed} in volume (combining together the sizes of individual tumours when more than one was present).
The tumour material was excised aseptically, then processed as described for primary tumour. 
Briefly, the tumour was harvested and minced finely with scalpels then mechanically disaggregated for one minute using a Stomacher 80 Biomaster (Seward Limited, Worthing, UK) in \SIrange{1}{2}{\ml} cold DMEM-F12 medium with Glucose, L-Glutamine and HEPES. 
Aliquots from the resulting suspension of cells and fragments were used for xenotransplants in next generation of mice and cryopreserved.
Serially transplanted aliquots represented approximately 0.1-0.3\%  of the original tumour volume. HER+ and TNBC PDX were passaged upto 10 generations and scWGS was carried out at each timepoint while scRNA-seq was done at initial, mid and late timepoint. 

\subsection{TNBC PDX tumour mixing experiments} Frozen untreated passages three (X3) and eight (X8) vials from TNBC PDX, were thawed and  physically remixed in two different volumetric proportions of X3:X8 by tumour weight at the ratio of approximately 1:1 and 1:0.4, labelled as mixture branch a and branch b, respectively. From each of different dilutions, \SI{200}{\ul} of aliquot was taken to be used to transplant in two mice each using the same protocol of transplantation as described above. Before transplantation a small proportion of the physical mixture of cells, from the 1:1 ratio, was subjected to whole genome single cell sequencing to measure the baseline clonal composition labelled as M0 and its subsequent PDX as M1 \textbf{(\autoref{fig:Mixturenew} a)}. Each of the thawed X3 and X8 cell populations used for mixing were also transplanted independently to confirm the viability of the tumour material for PDX tumour growth. The tumour cell mixture was then serially passaged over 4 generations, designating the transplants as M1-M4. Tumours from each X3:X8 serial passages from both mixture branches were collected and analysed with scWGS (DLP+) as for other samples.

\subsection{TNBC PDX timeseries treatment with chemotherapies}
\label{ssec:rx}
NRG mice of the same age and genotype as above were used for transplantation treatment experiments. Drug treatment with cisplatin (Platinum) was commenced when the tumour size reached approximately \SIrange{300}{400}{\mm\cubed}. Cisplatin (Accord DIN: 02355183)  was administered i.p. at \SI{2}{\mg\per\kg} every third day for 8 doses maximum (Q3Dx8). The dosage schedule was adjusted 50\% less than what is mentioned in the literature \cite{li2013enhanced,wang2013klotho} and 50-60\%  of the maximum tolerated dose (MTD) calculated in the immunodeficient mice  \textbf{(\autoref{fig:treatmentdesignMTD} b)} . Low dose cisplatin pulse and tumour collection timings were optimized to achieve the experimental aims of tumour resistance. The aim was to collect tumour at 50\% shrinkage (from the starting tumour at the time treatment started) in size when measured with a caliper. Cisplatin \SI{1}{\mg\per\ml} was diluted in 0.9\% NaCl to obtain concentrations \SI{200}{\ul}/\SI{20}{\g} of mouse weight and kept in glass vials at room temperature. Quality control (QC) samples were prepared freshly on each day prior to the dosing. Mice were continually monitored for acute signs of toxicity including pain at injection site, skin tenting, coat scruffing, sunken eyes, food consumption and behaviour for the first two hours following compound administration. For all TNBC PDX, 8 mice at initial passage were transplanted in parallel for the treatment/treatment holiday study group. Half of the mice were treated with cisplatin when tumours exhibited around 50\% shrinkage, the residual tumour was harvested as above and re-transplanted for the next passage in the group of eight mice. Again, half of the mice at X5 were kept untreated while the other half were exposed to cisplatin following the same dosing strategy. Four cycles of cisplatin treatment were generated, with a parallel drug holiday group at each passage. Cisplatin treated tumours were coded as \textit{UT, UTT, UTTT, UTTTT} for each of the four cycles of drug respectively, while the tumours on drug holiday were labelled as \textit{UTU, UTTU} and \textit{UTTTU} for the three timepoints. Each number of \textit{T} exhibits the number of cycles of drug exposure \textbf{(\autoref{fig:treatmentdesignMTD} a)}. 

SA535 TNBC PDX was also treated with CX5461, the same way like cisplatin. CX5461 was dissolved in 50mM NaH2PO4, pH4 for xenograft application. CX5461 was administered through oral gavage once every three days at the final dose of \SI{50}{\mg\per\kg}, which is less than the calculated MTD of \SI{65.2}{\mg\per\kg}.

In particular, TNBC-SA609 PDX was processed to establish 5 independent lines. All five lines from TNBC-SA609 were passaged identically after initial establishment. Line 1 untreated samples were seeded (X3 to X4) from a freshly dissociated tumour, whereas 4 other lines (all treated and line 2 Un-treated) were seeded (X3 to X4) from a cryopreserved vial of tumour (\textbf{\autoref{fig:schematic} a}). Technical replicates were collected and sequenced for lines 1, 2. 

ScWGS and scRNA-seq was carried out from each tumour during the timeseries treatment with counterpart drug holiday and untreated controls.

\begin{figure}
\centering
\includegraphics[width=\textwidth]{Figures/chap2/treatmentdesignMTD.pdf}
	
\caption[Experimental overview of TNBC PDX treated time series]
	{\small
	\textbf{Experimental overview of treatment design in TNBC PDX time series}
\textbf{(a)} Experimental design of cisplatin treatment in PDX.The residual tumour from one treated mouse was re-transplanted in the next (n=8). The solid blue colour represent cisplatin treated tumours \textit{(UT, UTT, UTTT, UTTTT)}; blue outlined in grey represents drug holiday \textit{(UTU, UTTU, UTTTU)}. Grey represent the untreated series \textit{(U, UU, UUU, UUUU, UUUUU)} \textbf{(b)} Mouse body weight graph recorded during \ac{MTD} evaluation of cisplatin in NRG mice (n=3 in each study cohort).}
	
	\label{fig:treatmentdesignMTD}
\end{figure}

\subsection{PDX tumour growth measurement curves} 
NRG mice received sub-cutaneous inoculation (SQ) of tumour cells (\SI{150}{\ul}) on day 0. 
The tumours were allowed to grow to palpable solid nodules.
Around 7-9 days after they were palpable, their size were measured with calipers every 3rd day. 
Tumours were measured in two dimensions using a digital caliper and expressed as tumour volume in mm3; defined as: [volume= 0.52$\times$(Length)$\times$(Width)].
Tumour growth inhibition (TGI) percentage range was defined as: [1 - (mean volume of treated tumours)/(mean volume of control tumours) x 100\%] \cite{hather2014growth}.

\section{PDX tumor dissociation into single cells}

\subsection{Tissue dissociation at 37\textdegree C}
Tumour fragments from patient breast and ovarian samples and PDX were incubated for 2 hrs with a collagenase/hyaluronidase enzyme mix in serum-free Dulbecco's Modified Eagle's Medium (DMEM) at 37\textdegree C with intermittent gentle trituration with a wide bore pipette tip. Cells were resuspended in 0.25\% trypsin-edta for 1 min followed by neutralization with 2\% FBS in Hank's Balanced Salt Solution (HBSS) and centrifugation. Cells were resuspended in 2\% FBS/HBSS and filtered through a 40\textmu m filter. Where necessary, dead cells were removed using MACS Dead Cell Removal Beads (Miltenyi Biotec) according to the manufacturer's instructions. Cells were centrifuged and resuspended in 0.04\%  BSA/PBS and cell concentration adjusted for scRNA-seq. For timecourse experiment, tissue was dissociated as above for 3 hrs with samples taken at 30 mins, 1 hr and 2 hr.

\subsection{Tissue dissociation at 6\textdegree C}
Tumour fragments were incubated for 30 minutes at 6\textdegree C with a serine protease, subtilisin A, derived from the Himalayan soil bacterium \textit{Bacillus Lichenformis} (Creative Enzymes NATE0633) in PBS supplemented with 5mM CaCl2 and 125U/ml DNAse, as described in \cite{adam2017psychrophilic, potter2019dissociation}. During dissociation, samples were gently triturated every 5 minutes using a wide-bore pipette. Cells were resuspended in 0.25\% trypsin-edta for 1 minutes at room temperature, neutralized with 2\% FBS in HBSS and filtered through a 40\textmu m filter. Following dissociation, samples were processed for scRNA-seq as described above. For the timecourse experiment, tissue was dissociated as above for 3 hours with samples taken at 30 minutes, 1 hour and 2 hours. 

\subsection{Removal of murine contamination from patient derived xenograft samples}

To identify murine cells in the PDX samples, we re-ran CellRanger version 3.0.2 aligning cells to both GRCh38 and mm10 (separately). We then considered all cells for which a valid barcode was identified in the raw (unfiltered) data for either alignment, and counted the number of reads mapping to each genome for each cell. A cell was subsequently designated as a contaminating mouse cell if more reads mapped to mm10 than GRCh38, and a human cell otherwise.

\section{\textit{in vitro} cultures and assays}

\subsection{WST-1 assay}
In all the experiments in \textbf{\autoref{fig:invitro}}, the cell viability was measured by WST-1 assay according to the protocol provided by manufacturer (Roche, Catalogue number 11644807001). Single cells from PDX and Wild type and HCT116 (DNA-PK and Lig4 knockout) cells were plated in 96-well plates either untreated or treated with drug continuously during the indicated time before WST-1 assay. Drug incubation time was 6-7 days. Absorbance was read with the microplate reader SpectraMax 3 (Molecular Devices, Sunnyvale, CA, USA).


\section{Live/dead cells analysis}

\subsection{Flow Cytometry for live/dead cells staining}
 GM18507 cells were treated with or without 100ng/ml TNF\textalpha  for 24 hours before being stained with propidium iodide (Catalogue no.P1304MP, Theromofisher scientific) and FITC annexin V (Catalogue no.640905, Biolegend) and sorted into dying, dead or live populations according to single, double or negative staining respectively using a FACS Aria Fusion (BD Biosciences).
 
\subsection{Clustering of live, dying, and dead cells}
Cells were hierarchically clustered using the \texttt{hclust} function in \texttt{R} applied to the 10-dimensional output of mutual nearest neighbor (MNN), and clusters assigned using the \texttt{cutree} function.

\subsection{Reproducible data analysis}
A dockerized workflow to enable reproduction of all figures and analysis of chapter 3 \textit{Genome Biology} paper is available (Campbell, K. kieranrcampbell/scrnaseq-digestion-paper. Github. \url{https://github.com/kieranrcampbell/scrnaseq-digestion-paper}(2019). Corresponding docker image is at \url{https://cloud.docker.com/u/kieranrcampbell/repository/docker/kieranrcampbell/statgen2} (version 0.4).

\section{Single cell whole genome sequencing and library construction with DLP+}

\subsection{Creation of single cell suspensions from PDX}
Tumour fragments from PDX samples were incubated with a collagenase/hyaluronidase 1:10 (10X) enzyme mix (STEM CELL technologies, catalogue \#07912) in  \SI{5}{\ml} DMEM/F-12 with Glucose, L-Glutamine and HEPES (Lonza 12-719F) and 1\%BSA at \SI{37}{\degreeCelsius} with intermittent gentle pipetting up and down the sample every 30 min for 1 min, during the first hour with a wide bore pipette tip, and every 15-20 min for the second hour, followed by  centrifugation (1100 rpm, 5 min) and supernatant removal.
The tissue pellet was resuspended in \SI{1}{\ml} of  0.25 percent trypsin-EDTA (VWR CA45000-664) for 1 min, superadded by \SI{1}{\ml} of DNAse 1/dispase \SI{100}{\ul}/\SI{900}{\ul} (StemCell 07900,00082462) pipetted up and down 2 min, followed by neutralization with 2\% FBS in HBSS with 10 mM HEPES (STEMcells Catalogue \#37150). 
This cell suspension was then passed through a \SI{70}{\micro\metre} filter to remove remaining undigested tissue and centrifuged for 5 min at 1100 rpm after topping it up to \SI{5}{\ml} with HBSS.
Single cells pellet  was resuspended in PBS + 0.04\% BSA (Sigma) in appropriate volume to achieve  $\sim$~1 million per ml concentration of cells for robot spotting for DLP+.

\subsection{Dead Cell removal method}
The final single cell single prep from above was centrifuged at 1100 rpm for 5 minutes and MACS Miltenyi Biotec Dead Cell Removal Kit (Catalogue no. 130-090-101) was used according to the manufacturer's protocol. Briefly, after complete removal of supernatant, cell pellet was resuspended in 100 ul of
Dead Cell Removal MicroBeads per million of cells, mixed and incubated at room temperature for 15 min. MS columns (Catalogue no. 130-042-201) were attached to OctoMACS magnetic separator (Catalogue no. 130-042-109) and primed by rinsing with 500 ul of 1X binding buffer that comes with the kit. Cell suspension was
applied to the column and washed 2 times with 500 ul of 1X Binding buffer. Effluent containing primarily live cells was collected, centrifuged at 1100 rpm for 5 min, resuspended in cold 0.04\% BSA/PBS and counted manually using hemacytometer.


\subsection{Robot spotting of single cells into the nanolitre wells and library construction}
 Single cell whole genome sequencing (scWGS) DLP+ library construction was carried out as described in \cite{laks2019clonal}. Briefly, single cell suspensions from cell lines and PDX were fluorescently stained using CellTrace CFSE (Life Technologies) and LIVE/DEAD Fixable Red Dead Cell Stain (ThermoFisher) in a PBS solution containing 0.04\% BSA (Miltenyi Biotec 130-091-376) incubated at \SI{37}{\degreeCelsius} for 20 minutes. Cells were subsequently centrifuged to remove stain, and resuspended in fresh PBS with 0.04 percent BSA. This single cell suspension was loaded into a contactless piezo electric dispenser (sciFLEXARRAYER S3, Scienion) and spotted into the open nanowell arrays (SmartChip, TakaraBio) preprinted with unique dual index sequencing primer pairs. Occupancy and cell state were confirmed by fluorescent imaging and wells were selected for single cell copy number profiling using the DLP+ method \cite{laks2019clonal}. Briefly, cell dispensing was followed by enzymatic and heat lysis. After cell lysis, tagmentation mix 
(\SI{14.335}{\nano\liter} TD Buffer, \SI{3.5}{\nano\liter} TDE1, and \SI{0.165}{\nano\liter} 10\% Tween-20) in PCR water were dispensed into each well followed by incubation and neutralization. Final recovery and purification of single cell libraries was done after 8 cycles of PCR. Cleaned up pooled single-cell libraries were analyzed using the Aglient Bioanalyzer 2100 HS kit. Libraries were sequenced at UBC Biomedical Research Centre (BRC) in Vancouver, British Columbia on the Illumina NextSeq 550 (mid- or high-output, paired-end 150-bp reads), or at the GSC on Illumina HiSeq2500 (paired-end 125-bp reads) and Illumina HiSeqX (paired-end 150-bp reads). The data was then processed to quantification and statistical analysis pipeline \cite{laks2019clonal}.

\section{Single cell whole genome data analysis}
\subsection{DLP+ sequence analysis, copy number determination and quality control filtering}
FASTQ pre-processing, sequence alignment, quality control, copy number calling, S-phase classification and filtering was performed on all libraries as described in \cite{laks2019clonal}. Briefly, cells were assigned a quality score (QS) for data quality based on a 13 feature Random forest classifier fitted and applied as per \cite{laks2019clonal}. Copy number alterations on a per cell basis were determined using a hidden Markov Model (HMM) approach using the \texttt{HMMCopy} package with parameterizations detailed in \cite{laks2019clonal}. S-phase cells were identified using an automated classifier trained using cell-cycle sorted cells and where features from HMM output were used to identify cells most probably in early or late phase replication of their genomes. As these cells interfere with downstream phylogenetic analysis, they were removed from the analysis according to parameter settings and thresholds established in \cite{laks2019clonal}. To further enable phylogenetic inference, 10-15\% of cells with highest average copy number state jumps were removed. Upon inspection, these cells included early and late dividing cells that were not captured by the s-phase classifier. Mouse cells were filtered from xenograft libraries using fastqscreen \cite{wingett2018fastq}.
 
 %is a Bayesian model and associated inference algorithm based on a diffusion approximation to K-allele Wright-Fisher model with selection. We start with timeseries clonal abundance measurements over a fixed number of clones and estimate two key unknown parameters of interest. Fitness coeffcients represents a quantitative measure of the growth potential of a given clone; and distributions over continuous-time trajectories, a latent (unobserved) population structure gives trajectory in `generational' time.
\subsection{Statistical models to infer evolutionary history of clones}
The two key statistical models used to measure and infer clonal evolution are: 
 
 (i) Bayesian phylogenetic inference method, to identify biologically related groups of cells (i.e., clones) across multiple timepoints. 
 In this dissertation, we applied newly developed phylogenetic model, called \texttt{sitka} \cite{dorri2020efficient}. \texttt{sitka} scales to thousands of cells for copy number states in scWGS. This is accompanied with a modified tree cutting algorithm, named as \texttt{Lumberjack}.

(ii) Quantitatively reason about the underlying evolutionary
forces acting on the clones through their observed dynamics
 using Bayesian state space \texttt{fitClone}  based on the Wright-Fisher diffusion with selection.

\subsubsection{Identifying clones through phylogenetic (\texttt{sitka}) analysis \cite{dorri2020efficient, salehi2020single}}
\texttt{sitka} is a single cell Bayesian tree reconstruction method, based on copy number change point binary variables \cite{dorri2020efficient} to fit phylogenetic trees to the copy number profiles. Using \texttt{sitka} the evolutionary relationships of cells were established in timeseries PDX heterogeneous samples. 
\\
\textbf{- Assumptions:} 
\\
\textbf{(1)} \texttt{sitka} assumes binary encoding of features (0 denotes loss, 1 denotes gain). The features include change in copy number points (breakpoints). 
\\
\textbf{(2)} Following the rules of Perfect persistent phylogeny, which assumes that all internal nodes are labelled such that all events evolve down without homoplasy. 
\\
\textbf{(3)} Phylogenetic trait arises exactly once on rooted tree topology. All individual cells descending from that position will inherit that trait (exactly one gain or loss) with no back mutations.
\\
\textbf{(4)} The phylogenetic tree is assumed to comprise internal nodes that are phylogenetic markers (loci) and terminal nodes that are either cells or loci
\\
\textbf{- \texttt{sitka} \textbf{input:} }
\\
\texttt{sitka} takes uses CNA-change points, i.e. markers  (breakpoints of copy number segments) instead of integer CNA states as input (phylogenetic traits) \cite{dorri2020efficient}. The binary  values, 0 and 1 are considered as loss and gain, respectively.
\\
\textbf{- \texttt{sitka} \textbf{output:} }
\\
In the output of \texttt{sitka}, clones are the terminal leaf nodes of the phylogenetic topology.  
This model first obtained a coarse tree structure from the reconstructed phylogeny by two step procedure. (i) First, identified  monophyletic clades via algorithm \cite{salehi2020single}.
(ii) Then removed the cells comprising the clades found in step one from the tree and repeated the algorithm. The new clades (if any) could be paraphyletic. Then, the fine structures within the initial clades were identified by traversing the tree in a bottom up manner and merging loci that are sufficiently similar.
The distance between two subclades was computed as the mean absolute difference of their median genotypes.
The degree of homogeneity can be tuned by limiting the number of loci and the difference in copy number of sub-clades in a clone. 
\\
The inferred trees were post-processed to identify clonal populations from major clades. With clonal populations defined, their abundances were counted as a function of timeseries and these were used for fitness inference (see below). 
\\
Clones were constructed by identifying connected components (each a clade or a paraphyly) in the phylogenetic tree reconstruction. The tree was `cut' (\texttt{Lumberjack}) into discrete populations  \cite{salehi2020single}.
\\
\textbf{- \texttt{sitka} \textbf{performance:} }
The performance of phylogenetic model could be confirmed by generating a SNV tree and compare the breakpoint tree with the SNV tree as in \cite{laks2019clonal}. 



%There are specific CNA events that will result in the violation of the perfect phylogeny assumption (PPA),
%for example, if a deletion event is followed by another deletion event that overlaps either end of the first event (detail in introduction).

%To investigate cancer evolution it is required to determine the abundance of subpopulations over time. To this end, \texttt{Lumberjack} was introduced, a tree-cutting algorithm that was used to define clonal subpopulations. Once clones were identified, the abundance of each clone at a specific timepoint was calculated as the fraction of cells in that clone from that timepoint.

%In the output tree of \texttt{sitka}, cells are part of the terminal leaf nodes of the phylogenetic topology. The inferred trees were post-processed to identify clonal populations from major clades. When clonal populations are defined, their abundances were counted as a function of timeseries and these were used for fitness inference. 
%Clones were constructed by identifying connected components (each a clade or a paraphyly) in the phylogenetic tree reconstruction. The tree was `cut' into discrete populations.


%\textbf{Phylogenetic tree inference, clone determination and clonal abundance measurements \cite{salehi2020single}.}


%%Notation

%Let $L$ be a set of loci and $C$ a set of cells and ${\lvert}L{\rvert}$. 
%Define $\tau = (L, C, E)$ to be a rooted phylogenetic tree with $E$ its set of directed edges.
%The phylogenetic tree is assumed to comprise internal nodes that are phylogenetic markers (loci) and terminal nodes that are either cells or loci. 
%Terminal loci are considered unused phylogenetic markers and are discarded. 
%Let ${\lvert}\tau{\rvert} = {\lvert}C{\rvert}$, that is the number of cells that belong to tree $\tau$.
%By $\tau_{l} = (L_l, C_l, E_l)$ denote the subtree rooted at node $l$.
%Set $\text{pa}(l)$ and $\text{child}(l)$ be the parent node and set of immediate children of node $l$ with $\text{desc}(l)$ comprising all its descendant. 

% Final clone determination
%The inputs to the algorithm are the rooted phylogenetic tree $\tau$ and the copy number states of its cells.
%A clone is defined as connected components (each a clade or a paraphyly) in the graph tree $\tau$ composed of cell of sufficient genomic homogeneity. 
%The degree of homogeneity can be tuned by limiting the number of loci and the difference in copy number of sub-clades in a clone. 
%The algorithm works by first finding the coarse structure, that is dividing the tree into major clades and then looking for fine structures within each clade by traversing the tree in a bottom up manner and merging loci that are sufficiently similar.
%The remaining loci constitute the roots of detected clades.


\subsubsection{\texttt{fitClone}-quantitative measure of fitness model}
 
 \texttt{fitClone} is a statistical model which determines whether measurements across a timeseries, that show growth or decline of cell populations, could have arisen by chance.  The basic intuition of Wright-Fisher model is that small natural fluctuations in transmission of alleles during population growth could eventually lead to fixation of one of the subpopulations. 
 In population genetics this is referred to as genetic drift. In the cancer system under study in this dissertation, its application allows to test for fluctuations in subpopulation growth and decline that could be neutral, as opposed those associated with non-neutral fitness. 
\\
\textbf{- \texttt{fitClone} \textbf{input:} } 
 	\begin{itemize}
 	    \item 	Number of clones and number of timepoints.
    \item 	A reference clone, for example, clone C in \textbf{\autoref{fig:fitCloneinterpretation}}.
 	   \item The abundance of each clone at each timepoint (timeseries clonal abundance data)
 	   \item The effective population size (according to WF the population is fixed), the number of cells in an idealised system.
	\end{itemize}
\
\textbf{- \texttt{fitClone} \textbf{output:} } 
 	\begin{itemize}
 	    \item 	The selection coefficient for each clone, with the reference clone's selection coefficient set to zero (0).
		\item	The imputed clonal abundances in intermediate unobserved timepoints.
\\ 
 \textbf{- Interpretation of \texttt{fitClone} } 
 From observed clonal abundance at each time point, clonal frequencies were inferred \textbf{\autoref{fig:fitCloneinterpretation}}. Selection coefficients were calculated by adding 1 to the selection coefficient value. The reference clone is considered as zero. Fitness is always relative to something and in this example clone C is taken as reference clone. In the example given, clone E has the highest fitness as it has highest selection coefficient value.
\\
\textbf{- \texttt{fitClone} \textbf{performance:} } 

Multiple strategies were used to confirm performance of fitClone:
\begin{itemize}
	
	\item Predicted trajectories were expected to be in line with observed ones.
	
	\item Repeating the same experiments by diluting the proportions of high fitness with low fitness clones and let them compete. Clones with higher estimated `s' were more likely to rise in the mixture, even when started from a low initial fraction.

\end{itemize}
Framing the interpretation of the results in terms of prediction may also be helpful:
\\
If a clone has a higher `s' value than another clone, then starting from the same starting population size, it is more likely that former would dominate the latter



 
 
 
 
 
 

%----------------------------------------------------------------------------------
 	\begin{figure}
	\centering
	\includegraphics[width=\textwidth]{Figures/chap2/fitcloneinterpretation.png}
	\caption[Interpretation and example of \texttt{fitClone} ]
	{\small
	    \textbf{Interpretation and example of \texttt{fitClone}.}
	    \textbf{(Left panel) Observed clonal fraction}. Vertical axis shows clonal fraction while horizontal axis shows real timepoints in days. Alphabets represent each clone name.  \textbf{(Right-top)}  Vertical axis shows clonal fraction and growth rate. Horizontal axis shows diffusion time,where Ne stands for effective population size. \textbf{(Right-Bottom)} Credible intervals for fitness coefficients. Vertical axis shows clone names. Horizontal axis shows the values of selection coefficient with the name of clone taken as reference to compare other clones.
	    
	}
	\label{fig:fitCloneinterpretation}
\end{figure}  
	\end{itemize}
%-----------------------------------------------------------------------------------


\section{Single cell RNA sequencing (scRNA-seq)}

\subsection{Processing of patient derived xenografts for scRNA-seq data}
PDX tumours were harvested and mechanically disaggregated into small fragments to viably freeze them according to the protocol mentioned above. 
One of the viable frozen tumour vial was thawed and after washing out the freezing media, the tumour clumps and fragments were incubated with digestion enzymes as with DLP+ preparation. After complete dissociation with collagenase/hyaluronidase enzyme mix according to the protocol at \SI{37}{\degreeCelsius}, followed by briefly washing in 0.05\% trypsin-EDTA and resuspension in 0.04\% BSA in PBS. Dead cells were removed using the Miltenyi MACS Dead Cell Removal kit and cells were processed as previously described \cite{o2019dissociation}.
To avoid processing artifacts, dissociation methods and times were tightly controlled and for treatment and treatment holiday pairs, library construction was performed on the same chips. Single cell suspensions were loaded onto the 10x Genomics single cell controller and libraries prepared according to the Chromium Single Cell 3â€™ Reagent Chemistry kit standard protocol. 
Libraries were then sequenced on an Illumina Nextseq500/550 with 42bp paired-end reads, or a HiSeq2500 v4 with 125bp paired-end reads. 10x Genomics Cell Ranger version 3.1 was used to perform demultiplexing,  counting and alignment to human genome (GRCh38).

\subsection{Quality control for scRNA-seq data}

Count matrices were generated using CellRanger version 3.0.2 (V3 chemistry). Cells were considered to have passed a quality control filter (QC-filter) and retained for subsequent analysis if they met the following criteria: (i) at least 1000 genes detected, (ii) less than 20\% of counts (UMIs) mapping to genes from the mitochondrial genome (``mitochondrial genes''), (iii) fewer than 60\% of counts (UMIs) mapping to ribosomal genes, and (iv) the total counts (UMIs) per cell was at most 3 median absolute deviations lower than the overall median. Cells not matching all criteria were filtered using the \texttt{calculateQCMetrics} and \texttt{isOutlier} functions in the \texttt{scater} package \cite{mccarthy2017scater}. Then, all the mouse cells were eliminated. A cell is called a mouse cell if the total number of counts in a mouse alignment of the 10x sample was greater than the total number of counts in a human alignment. Finally, we eliminated doublets using package \texttt{scrublet} \cite{scrublet}.

\subsection{Gene expression normalization}

Sample level normalized log expression values were computed using twice \texttt{scran} \cite{lun2016pooling} with grouping variables calculated from clustering using the \texttt{quickCluster} function \cite{lun2016step}. Overall normalized expression was computed by merging sample level matrices and recalculating normalized expression with grouping labels computed on the merged matrix. Batch correction between libraries was performed using Scanorama \cite{hie2019efficient} on the overall normalized expression values with default parameters and the library id as the batch label. 

\subsection{Dimensionality reduction}

Principal component analysis was performed on the batch corrected expression matrix using the \texttt{scater} \cite{mccarthy2017scater} R package. A single two dimension UMAP \cite{becht2019dimensionality} embedding was generated using the first 50 principle components. This embedding was used in downstream analysis and visualizations.

\subsection{Pathway Enrichment Networks}

Enriched pathways were computed from differentially expressed genes (adjusted p-value $<$ 0.01) ranked by log fold change.  A normalized enrichment score (NES) was calculated from a ranked gene set enrichment analysis (GSEA) \cite{shi2007gene} performed on each subset of differentially expressed genes using the hallmark gene set collection from MSigDB \cite{liberzon2015molecular}.  Significantly enriched pathways (adjusted p-value $<$ 0.01) and pathway specific differentially expressed genes were included in network enrichment figures.  
Pathway nodes were colored by NES value. Edges are defined between pathways sharing genes.
All analysis and visualization was performed using \texttt{gseapy} and \texttt{networkx} \cite{hagberg2008exploring} Python packages.

\subsection{Integrative genome-transcriptome analysis with \texttt{clonealign}}
\texttt{clonealign} version 1.99.2 was used to align scRNA-seq cells to the DLP+ clones obtained. First, genes whose purity was less than 60\% in any clone (40\% for \textit{X5 UT}) were removed, where the purity of a gene in a clone  is defined as the percentage of cells that have the modal copy number for that gene and clone. Then, we used the following clonealign parameters: \texttt{n\_repeats = 3}, \texttt{mc\_samples = 1}, \texttt{learning\_rate = 0.07}, \texttt{max\_iter = 500}, \texttt{data\_init\_mu = TRUE} for all samples except \texttt{FALSE} for \textit{X5 UTU}, \texttt{data\_init\_mu = FALSE},
\texttt{saturation\_threshold = 6} and \texttt{clone\_call\_probability = 0.9}.

\section{Differential expression analysis}

\subsection{Differential expression of temperatures and core heat-related gene set} 

All differential expression analyses were performed with edgeR \cite{robinson2010edger} version 3.24.3 using the quasi-likelihood F-test as was the top-performing method in a recent review \cite{soneson2018bias}. The PDX or cell line ID in the design matrix to account for unwanted technical and biological variation. In every case we only considered genes with minimum 10 counts across all cells.
The core set of genes were defined as those with FDR adjusted Q-value $<0.05$ and with $|\log_2(\text{fold change})| > \log2(1.5)$, in other words, the average change in expression was required to be either 50\% greater or less than the baseline to include the gene. Overall this gave 192 genes (182 upregulated and 10 downregulated).
Pathway enrichment was performed using camera \cite{wu2012camera} with \texttt{trend.var=TRUE} on the Hallmark gene set \cite{liberzon2015molecular} retrieved from \url{http://bioinf.wehi.edu.au/software/MSigDB/human\_H\_v5p2.rdata} with timestamp 2016-10-10.

Differential expression for the digestion enzyme vs. time comparisons were performed as above. Only pairwise comparisons were considered, e.g. for the 2 hour vs 30 minute collagenase only comparison, the dataset was subsetted to contain only these cells and differential expression analysis was performed.

\subsection{Differential expression analysis between resistant and sensitive clones}
Differential expression quantifiers including log2 fold change and FDR  were computed using the R 3.6.0 Bioconductor package \texttt{edgeR\_3.26.0} that implements scRNA-seq differential expression analysis methodology based on the negative binomial distribution. First call the \texttt{estimateDisp} function was called to estimate the dispersion by fitting a generalized linear model that accounts for all systematic sources of variation. Next,  \texttt{edgeR} functions \texttt{glmQLFit} and \texttt{glmQLFTest}, were used to perform a quasi-likelihood dispersion estimation and hypothesis testing that assigned false discovery rate values to each gene. In the track and volcano plots, a positive log2 fold change value for resistant clone relative to sensitive clone signified that the gene was significantly upregulated (at a given FDR threshold).  Similarly, a gene with negative log2 fold change was significantly downregulated in resistant than in sensitive clone.

\subsection{Selection of house keeping genes}
To use reference house keeping (HK) genes as indicators in timeseries PDX scRNA-seq data,  a custom list of HK genes were selected, based on the reference HK gene set \cite{lin2019evaluating}. Then, the most stable HK genes with the stability indexes  greater than 75\% quantile were selected. Next, the variance level of stable HK genes were estimated by quantifying the average of HK genes in the dataset. That resulted in a range of variance of 0.05 to 0.4. The HK genes with low variance, with the range of 0 to 0.2 in the data were kept, and above that threshold were dropped. From this procedure, a list of the custom most stable HK genes was obtained and used as an indicator to test batch effects. 


\subsection{Linear Model fit}
Linear model fits for each condition (untreated or treated) were obtained using the \texttt{lm} function in R, where discrete time points were used on the x axis (for example 1, 2, 3 for X4, X5, X6), and the mean \texttt{SCTransform} log2 normalized gene expression at all the available time points for that condition was used on the y axis.
The slope of the linear fitting (Y $\sim$ X) signified the overall magnitude of change from one time point to the next. Since the y axis was on a log2 scale, a slope of 1 means that, on average over time, the expression level doubled with every time point. Similarly, slopes 0.1 and 0.2 mean that, on average, the expression level increased by 7\% and 15\%, respectively, at every time point.

\section{Other statistical methods}
Data was analyzed using GraphPad Prism 8 (GraphPad Software) and R version 3.2.0.