   %% The following is a directive for TeXShop to indicate the main file
%%!TEX root = diss.tex

\chapter{Preface}
This thesis consists of an introduction, materials and methods, three research chapters, and discussion followed by bibliography.

All animal experiments were conducted according to the Canadian Council on Animal Care guidelines, approved by the Animal Care Committee of the University of British Columbia, according to the guidelines of the Institutional Animal Care and Use Committee (IACUC) under protocols A14-0290 and A15-0248.
%-------------------------------
  
  \textbf{(i) Chapter 3:}

All experiments in this chapter were designed, conducted and analysed by me, unless stated otherwise, with intellectual inputs from Dr. Sam Aparicio. PDX tumors palpation were done by animal technicians of Aparicio lab, Ms. Teresa Ruiz de Algara and Ms. So Ra Lee.


From the drug efficacy \textit{in vitro} cultures, few experiments contributed to the publication of \textit{Nature Communication, 2017} paper \cite{xu2017cx}:

\textbf{CX-5461 is a DNA G-quadruplex stabilizer with selective lethality in BRCA1/2 deficient tumours.}

Xu H, Di Antonio M, McKinney S, Mathew V, Ho B,O'Neil NJ,Santos ND,Silvester J, Wei V, Garcia J, \emph{\textbf{Kabeer F}}, Lai D, Soriano P, Ban√°th J, Chiu DS, Yap D, Le DD, Ye FB, Zhang A, Thu K, Soong J, Lin SC, Tsai AH, Osako T, Algara T, Saunders DN, Wong J, Xian J, Bally MB, Brenton JD, Brown GW, Shah SP, Cescon D, Mak TW, Caldas C, Stirling PC, Hieter P, Balasubramanian S, Aparicio S. Nat Commun. 2017 Feb 17;8:14432, PMID: 28211448.



 \textit{In vivo} drug sensitivity experiments were all done by me including, tumor transplantation, drug dosing, monitoring, tumor measurement and tumor measurements records. Growth curve graphs were generated with help from Steven McKinney.


The other half of chapter 3 is based on \textit{Genome biology} paper \cite{o2019dissociation}:


\textbf{Dissociation of solid tumor tissues with cold active protease for single-cell RNA-seq minimizes conserved collagenase-associated stress responses.}

O'Flanagan CH*, Campbell KR*, Zhang AW*, \emph{\textbf{Kabeer F*}}, Lim JLP, Biele J, Eirew P, Lai D, McPherson A, Kong E,
Bates C, Borkowski K, Wiens M, Hewitson B, Hopkins J, Pham J, 
Ceglia N, Moore R, Mungall AJ,
McAlpine JN; CRUK IMAXT Grand Challenge Team, Shah SP, Aparicio S.

\textbf{*equal contribution.} Genome Biol. 2019 Oct 17;20(1):210, PMID: 31623682 

All patient derived xenograft (PDX)  and live/dead \ac{FACS} experiments for the \textit{Genome biology} paper was executed by me including tumor transplants, harvest, collagenase dissociation and FACS. Cold protease enzyme digestion was done by Ciara O'Flanagan and bioinformatic analysis were done by Kieran Campbell.

The established PDX tumors and dissociation optimizations done for chapter 3, also helped in 3 more publications from Aparicio lab, including \cite{laks2019clonal, campbell2019clonealign, de2020epiclomal}:
     
\textbf{Clonal Decomposition and DNA Replication States Defined by Scaled Single-Cell Genome Sequencing.}
Laks E, McPherson A, Zahn H, Lai D, Steif A, Brimhall J, Biele J, Wang B, Masud T, Ting J, Grewal D, Nielsen C,Leung S,Bojilova V,Smith M, Golovko O, Poon S, Eirew P, \emph{\textbf{Kabeer F}}, Ruiz de Algara T, Lee SR, Taghiyar MJ, Huebner C, Ngo J, Chan T, Vatrt-Watts S, Walters P, Abrar N, Chan S, Wiens M, Martin L, Scott RW, Underhill TM, Chavez E, Steidl C, Da Costa D, Ma Y, Coope RJN, Corbett R, Pleasance S, Moore R, Mungall AJ, Mar C, Cafferty F, Gelmon K, Chia S; CRUK IMAXT Grand Challenge Team, Marra MA, Hansen C, Shah SP, Aparicio S. Cell. 2019 Nov 14;179(5):1207-1221.e22,  PMID: 31730858.


\textbf{clonealign: statistical integration of independent single-cell RNA and DNA sequencing data from human cancers.}
Campbell KR, Steif A, Laks E, Zahn H, Lai D, McPherson A, Farahani H, \emph{\textbf{Kabeer F}}, O'Flanagan  C, Biele J, Brimhall J, Wang B, Walters P; IMAXT Consortium, Bouchard-Cote A, Aparicio S, Shah SP. Genome Biol. 2019 Mar 12;20(1), PMID: 30866997. 

\textbf{Epiclomal: Probabilistic clustering of sparse single-cell DNA methylation data.}
P E de Souza C, Andronescu M, Masud T, \emph{\textbf{Kabeer F}}, Biele J, Laks E, Lai D, Ye P, Brimhall J, Wang B, Su E, Hui T, Cao Q, Wong M, Moksa M, Moore RA, Hirst M, Aparicio S, Shah SP. PLoS Comput Biol. 2020 Sep 23;16(9):e1008270. doi: 10.1371/journal.pcbi.1008270. Online. PMID: 32966276

%-------------------------------

  \textbf{(ii) Chapter 4:}

This chapter is based on a manuscript under revision in \text{Nature}.


 \textbf{Single  cell  fitness  landscapes  induced  by  p53  deletion and platinum perturbation in cancer}

Sohrab Salehi*, \emph{\textbf{Farhia Kabeer*}}, Nick Ceglia, Mirela Andronescu, Marc Williams, Kieran R. Campbell, Tehmina Masud, Beixi Wang, Justina Biele, Jazmine Brimhall, Jerome Ting, Allen Zhang, Ciara O'Flanagan , Fatemeh Dorri, Nicole Rusk, Emma Laks, Hakwoo Lee, Teresa Algara, Sora Lee, Brian Yu Chieh Cheng, Peter Eirew, Takako Kono, Jennifer Pham, Diljot Grewal, Daniel Lai, Richard Moore, Andrew J. Mungall, Marco A Marra, IMAXT Consortium, Andrew McPherson, Alexandre Bouchard-Cote, Samuel Aparicio+, Sohrab P. Shah+

 \textbf{*equal contribution.} Under Review in \textit{Nature}, 2021

https://www.biorxiv.org/content/10.1101/2020.05.08.081349v1.full.pdf

Dr. Aparicio and Dr. Shah planned biological substrates, experimental designs and bioinformatic modeling. All PDX experiments were done by me from tumor serial transplants, drug dosing, monitoring, tumor collection and tumor dissociations. The single cell technicians in Aparicio lab spotted the single cells on chip and submitted the libraries to \ac{GSC}.
I receieved, organized and discussed the data for analysis with Sohrab Salehi. He did the main bioinformatic analysis on my data. Few other members of Dr. Shah lab, from Memorial Sloan Kettering (MSK), New York, also contributed in the analysis. 

%-------------------------------

   \textbf{(iii) Chapter 5:}
   
This chapter is based on the manuscript in preparation. All the PDX experiments and sample preparations are done by me. The single cell technicians in Aparicio lab prepared the 10X genomics libraries and submitted the samples to \ac{GSC}. All the bioinformatic analysis are done by Hoa Tran and Mirela Andronescu under guidance of Dr. Aparicio.
  
\textbf{Longitudinal tracking of drug-induced transcriptomic reprogramming and RNA velocity in triple negative breast cancer pre-clinical model}

\emph{\textbf{Farhia Kabeer*}}, Hoa Tran*, Mirela Andronescu*,   
Nicholas Ceglia, Hakwoo Lee, Sohrab Salehi, Beixi Wang, Justina Biele, Jazmine Brimhall, David Gee, Steven Mckinney, Ciara O'Flanagan, Teresa Algara, Takako Kono, Jennifer Pham, Daniel Lai, Richard Moore, Andrew J. Mungall, IMAXT Consortium,  Andrew  McPherson, Andrew Roth, Kieran R. Campbell, Sohrab P. Shah, Samuel Aparicio.