\begin{figure}
\centering
\includegraphics[width=0.9\linewidth]{figures/supp/sfig_simulations}
\caption{Simulations - Main claims}
\label{sfig:simul}
\end{figure}


%%%%%%%
\begin{figure}
\centering
\includegraphics[width=0.9\linewidth]{figures/supp/sfig_pdxExpDesCurves}
\caption{Overview of experimental design and PDX growth curves
(A)	In the top panel, the pink dishes represent the 184-hTERT L9 WT cell line 
being serially passaged. 
The shown passage numbers were subjected to single cell whole genome sequencing. 
Initial, mid, and late time points were selected for expression analysis and cells were sequenced at single cell RNA level on the 10X genomics platform. 
The below orange dishes represent home-made 184-hTERT L9 95.22; p53 null cell line by CRISPR technology. 
Parallel branches SA906a and SA906b were derived from the same tenth passage to measure the extent of heterogeneity observed in the same serially passaged baseline P53 null cells. 
Initial, mid and late timepoints were also selected from both the branches for single cell whole genome sequencing (DLP+) and single cell RNA sequencing for  expression analysis (10X genomics).(B)Western blot confirming knock out of Tp53 from 184-hTERT L9 WT cell line.(C) The first schematic is showing the sketch of the experimental strategy. The patientâ€™s breast tumours were taken and mechanically desegregated into small clumps and single cells. They were then re-transplanted into immuno-compromised mice to generate passage one (X1) of the patient derived xenograft (PDX). When the tumours reached the size of 1000 cubic millimeter, the tumours were harvested and again mechanically desegregated into tumour clumps and/or single cells. They were re-transplanted into a new set of mice to generate the next passage of PDX. Here, each big dark grey circle is representing the group of mice transplanted in each passage. Up to ten passages of PDX were derived by mechanical desegregation and serial re-transplanted of PDX tumors. The lower two schematics are showing the actual sampling done from SA609 in (TNBC) and SA532 (HER2+). The dark grey circles represent each mouse which was sampled for single cell whole genome sequencing. The light grey circles representing the replicates of tumour-bearing mice at the same time point.(D) This panel is showing the individual tumor growth from each passage of SA609 and SA532 PDXs. The vertical axis represent the tumor volume measured with calipers in cubic millimeters while the horizontal axis represents the time in days as the tumor grows.(E) H\&E and (IHC) immunohistochemistry of the two PDX tumors showed that SA609 is a derived from a triple negative breast cancer patient while SA532 was derived from a HER2+ breast cancer patient. HER2+ status on SA532 was also confirmed by RNA transcriptomes and FISH (Fluorescence in situ hybridization) analysis.}
\label{sfig:pdxExpDesCurves}
\end{figure}

%%%%%%%
\begin{figure}
\centering
\includegraphics[width=0.9\linewidth]{figures/supp/sfig_pdxmixrx}
\caption{Fitness validation with tumor mixing and drug perturbation (A)	The first schematic is showing the sketch of the mixing experimental strategy and drug perturbation. The frozen tumors from third and eighth passages were thawed at room temperature and mixed to create approximately equal population of each of them together in one sample.They were then transplanted in a new group of immuno-compromised mice to generate passage one (X1) of the mixed sample xenograft (PDX).Then serial passaging was done from this starting mixture sample and another line was propagated. (B) graph of mouse body weight recorded during maximum tolerated dose (MTD) evaluation of cisplatin in NRG mice (n=3 in each study cohort).(C) The top panel showing the experimental design of drug treatment in PDX.The residual tumor from one treated mouse was re-transplanted in the next (n=4). The lower panel showing the IDs assigned to each tumor sample during the course of treatment. The solid blue colour representing cisplatin treated tumors (UT,UTT,UTTT,UTTTT). while the blue outlined grey represent the tumors that were kept on drug holiday (UTU,UTTU,UTTTU). They were treated in the previous cycle of the drug but at that time point they were not exposed to the drug and kept as controls for that cycle.All grey are representing absolutely untreated branch (U, UU,UUU,UUUU,UUUUU). (D) Experimental and computational pipeline workflow showing enzymatic tumor digestion followed by robotic spotting and custom single cell direct library prep. (DLP+)and single cell RNA seq. After library construction pooled library is submitted to Genome Sciences Centre (GSC) for sequencing. Once the data is transferred by a file management system, Tantalus, it goes into the automation workflow analysis pipeline, Sisyphus.Then we ran our fitness model fitClone, on the final DLP+ data. (E) Tumor response curves in each cycle of cisplatin treatment.}
\label{sfig:pdxmixrx}
\end{figure}



\begin{figure}
\centering
%{Fitness validation with tumor mixing and drug perturbation (A) Schematic overview of clonal mixture experiment showing source samples from the original timeseries and serial propagation into a new line.  (B) Mouse body weight graph recorded during maximum tolerated dose (MTD) evaluation of cisplatin in NRG mice (n=3 in each study cohort).(C)  Experimental design of cisplatin treatment in PDX.The residual tumor from one treated mouse was re-transplanted in the next (n=4). The solid blue colour representing cisplatin treated tumors \textit{(UT,UTT,UTTT,UTTTT)}; blue outlined in grey represent drug holiday \textit{(UTU,UTTU,UTTTU)}. Grey represent the untreated series \textit{(U,UU,UUU,UUUU,UUUUU)}. (D) Experimental and computational pipeline workflow to generate DLP+ as previously described\cite{laks2019clonal} and 10X scRNAseq. DLP+ analysis was then fed to fitClone for model fitting. (E) Tumor response curves in each cycle of cisplatin treatment.}
\label{sfig:pdxmixrx}
\end{figure}
